common/OpTestHMC.py

```diff
@@ -55,6 +55,42 @@
 BOOTTIME = 500
 STALLTIME = 5
 
+class OpSecureBootUtilities():
+    '''
+    This class addresses the secureboot related pre-requisites.
+    '''
+
+    def __init__(self, console=None, distro=None):
+        self.console = console
+        self.distro = distro
+
+    def check_kernel_signature(self):
+        vmlinux = "vmlinuz"  # RHEL
+        if self.distro == 'SLES':
+            vmlinux = "vmlinux"
+
+        # Checking whether the kernel is signed or not
+        cmd = "strings /boot/%s-$(uname -r) | tail -1" % vmlinux
```

Comment:
a common practice is to not use pipe , instead parse the output in next step

Reply:
As per our discussion, since the rendered output is too huge, and the console being unable to handle the same, decided to continue using the piped tail command. Marking this as "Resolved"